### ISSUE TITLE
handle multi band land cover asset

### ISSUE THREAD

**Issue by i-m-amit** [12rambau/sdg_indicators_module#91]
Need to change the band renaming function to handle multi band land cover asset.

**Comment by 12rambau** [comment]
are talking about the reclassifyer itself ?

**Comment by i-m-amit** [comment]
No, suppose we don't need to reclassify the landcover and that can be used directly into the module and has more than one bands then it results in an error.

**Comment by 12rambau** [comment]
ok so I should just add a "band selector" next to the asset selector right ?

**Comment by i-m-amit** [comment]
yes.
